src/style.css

```diff
@@ -1 +1,61 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir-Book;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+.body {
+  margin: 0;
+}
+
+.container {
+  margin-top: 20px;
+  display: flex;
+  align-items: center;
+}
+
+.search___bar {
```

Comment:
The class name 'search___bar' contains three underscores, which might be a typo. Ensure consistency in class naming conventions.